superx/routing.py

```diff
@@ -4,3 +4,8 @@
 @app.route('/')
 def index_route():
     return home.home()
+
+@app.route('/login')
+def login():
```

Comment:
You might want to have a consistent naming scheme.
index is called "index_route", I would suggest either changing that to index or changing login to "login_route"

Reply:
Changed "index_route" into "index" for simplicity.